translations/de.json/de.json

```
@@ -0,0 +1,129 @@
+	"Créer": "Erstellen",
+	"Envoi en cours, veuillez patienter": "Upload in Bearbeitung, bitte warten",
+	"Votre contribution a bien été prise en compte, merci pour votre participation !": "Ihr Beitrag wurde berücksichtigt, vielen Dank für Ihren Beitrag!",
+	"Retour à la carte": "Zurück zur Karte",
+	"OK": "OK",
+	"Vous êtes connecté ; ce formulaire va éditer directement OpenStreetMap": "Sie sind eingeloggt; dieses Formular wird direkt OpenStreetMap bearbeiten",
+	"Vous n'êtes pas connecté ; ce formulaire va créer une note dans OpenStreetMap": "Sie sind nicht eingeloggt; Dieses Formular wird einen Hinweis in OpenStreetMap erstellen",
+	"Si vous quittez la page, les modifications en cours seront perdues.": "Wenn Sie die Seite verlassen, gehen die aktuellen Änderungen verloren.",
+	"Valider": "Validieren",
+	"Des données sont manquantes, merci de réessayer": "Daten fehlen, bitte versuchen Sie es noch einmal.",
+	"Une erreur s'est produite, merci de réessayer plus tard": "Ein Fehler passierte, bitte versuchen Sie es später noch einmal.",
+	"Objet concerné ;": "Betroffenes Objekt;",
+	"Plus d'infos sur la résolution de cette note": "Weitere Informationen über die Auflösung dieses Hinweises",
+	"Badges obtenus": "Abzeichen erhalten",
+	"Vous n'êtes pas connecté avec un compte OpenStreetMap. Inscrivez-vous ou connectez-vous pour collecter des badges !": "Sie sind nicht mit einem OpenStreetMap-Konto angemeldet. Registrieren Sie sich an oder melden Sie sich an, um Abzeichen zu sammeln!",
+	"Nouveau badge": "Neues Abzeichen",
+	"Voir tous les badges": "Alle Abzeichen anzeigen",
+	"Marqué comme résolu dans le cadre du projet du mois (via projetdumois.fr)": "Gekennzeichnet als gelöst im Rahmen Projekt des Monats (via projetdumois.fr)",
+	"Une erreur s'est produite lors de la modification de la note": "Bei der Änderung des Hinweises ist ein Fehler aufgetreten.",
+	"Une erreur s'est produite lors de l'enregistrement, merci de réessayer plus tard": "Bei der Registrierung ist ein Fehler aufgetreten, bitte versuchen Sie es später noch einmal.",
+	"Attributs": "Information",
+	"Attributs à renseigner": "Einzutragende Attribute",
+	"C'est fait": "Es ist erledigt.",
+	"Rien ici": "Hier ist nichts.",
+	"Intégrer cet objet": "Dieses Objekt einbetten",
+	"Vous devez être connecté pour marquer la note comme résolue.": "Sie müssen eingeloggt sein, um den Vermerk als gelöst zu markieren.",
+	"Participer à la discussion sur cette note": "Teilnahme an der Diskussion über diesen Vermerk",
+	"✘ Rien ici": "✘ Nichts hier.",
+	"Zoomez ou lancez une recherche pour commencer": "Zoomen Sie herein oder starten Sie eine Suche, um zu beginnen",
+	"Chargement en cours": "Laden in Arbeit",
+	"Déplacer la carte pour positionner l'objet à créer": "Karte verschieben, um das zu erstellenden Objekt zu positionieren",
+	"Afficher / masquer l'imagerie satellite": "Satellitenbilder anzeigen / ausblenden",
+	"Profil OpenStreetMap": "Profil OpenStreetMap",
```

I don't know the exact context, but probably this should be "OpenStreetMap-Profil" in German.
```suggestion
	"Profil OpenStreetMap": "OpenStreetMap-Profil",
```